demos/bridge/primefaces3-portlet/src/main/webapp/views/applicant.xhtml

```diff
@@ -66,7 +66,7 @@
 									<f:selectItem itemLabel="#{i18n['select']}" itemValue="" />
 									<f:selectItems itemValue="#{province.provinceId}" itemLabel="#{province.provinceName}"
 										value="#{listModelBean.provinces}" var="province" />
-									<p:ajax update="provinceIdField" />
+									<f:ajax render="provinceIdField" />
 								</h:selectOneMenu>
```

Comment:
@stiemannkj1 Please try changing h:selectOneMenu to http://www.primefaces.org/showcase/ui/input/oneMenu.xhtml (without f:ajax and without p:ajax) and if it works let me know when your branch is updated and then I'll merge.